libs/ha/ha_common_libs_k8s.py

```diff
@@ -257,18 +257,17 @@ def delete_s3_acc_buckets_objects(self, s3_data: dict, obj_crud: bool = False):
                         response = s3_del.delete_multiple_objects(_bucket, obj_list[1], quiet=True)
                         LOGGER.debug("Delete multiple objects response %s", response)
                 return True, "Successfully performed Objects Delete operation"
-            else:
-                for details in s3_data.values():
-                    s3_del = S3TestLib(endpoint_url=S3_CFG["s3_url"],
-                                       access_key=details['accesskey'],
-                                       secret_key=details['secretkey'])
-                    response = s3_del.delete_all_buckets()
-                    if not response[0]:
-                        return response
-                    response = self.s3_rest_obj.delete_s3_account(details['user_name'])
-                    if not response[0]:
-                        return response
-                return True, "Successfully performed S3 operation clean up"
+            for details in s3_data.values():
+                s3_del = S3TestLib(endpoint_url=S3_CFG["s3_url"],
+                                   access_key=details['accesskey'],
+                                   secret_key=details['secretkey'])
+                response = s3_del.delete_all_buckets()
+                if not response[0]:
+                    return response
+                response = self.s3_rest_obj.delete_s3_account(details['user_name'])
+                if not response[0]:
+                    return response
+            return True, "Successfully performed S3 operation clean up"
```

Comment:
Why this is changed?

Reply:
> Why this is changed?

This is just to fix pylint warning, as there is else after return statement giving the warning.